Xamarin/DrawXShared/RealmDraw.cs

```diff
@@ -338,7 +338,9 @@ private void DrawAPath(SKCanvas canvas, SKPaint paint, DrawPath drawPath)
         {
             using (var path = new SKPath())
             {
-                var pathColor = SwatchColor.ColorsByName[drawPath.color].Color;
+                // nasty hack because the Cocoa version defaults to Black
+                var pathColorName = (drawPath.color == "Black") ? "Charcoal" : drawPath.color;
```

Comment:
Assuming the model is unlikely to change, this was a once-off bug and apps shouldn't be generating unknown colours. Note that storing by name is actually more robust in case a given app gets the order wrong.